experimental/LieAlgebras/src/root_systems.jl

```diff
@@ -0,0 +1,198 @@
+mutable struct RootSystem
+  roots::Vector{Vector{Int}}
+  simple_roots::Vector{Vector{Int}}
+  positive_roots::Vector{Vector{Int}}
+  root_system_type::Tuple{String, Int64}
```

Comment:
And then here a symbol as well
```suggestion
  root_system_type::Tuple{Symbol, Int64}
```